### PR TITLE
Enhance ITMonitoringExporter tests by removing Domain resources after the tests completion

### PR DESCRIPTION
**Pull request by marinakog**
Refer to https://jira.oraclecorp.com/jira/browse/OWLS-92251

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7077

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterSideCar.java [hunk 1]
```diff
@@ -400,7 +401,9 @@ public void tearDownAll() {
           .withFailMessage("uninstallNginx() did not return true")
           .isTrue();
     }
-
+    shutdownDomain(domain1Namespace, domain1Uid);
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterSideCar.java:404]: Hi Marina, It doesn't help to shutdown the domains in the tearDown. These domains will be shutdown anyway once the IT class is done. We want to shut it down after the test method uses the domain and other test methods no longer needs it.

**Review by sankarpn — CHANGES_REQUESTED** [review]
Please see my comment.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterSideCar.java (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterSideCar.java]: fixed and rerun in https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/7091

**Review by anpanigr — COMMENTED** [review]
Remove the file ItMonitoringExporterSamples.java.new

**Review by anpanigr — COMMENTED** [review]
Remove the file  ItMonitoringExporterSideCar.java.new

**Review by anpanigr — COMMENTED** [review]
Remove ItMonitoringExporterWebApp.java.new

**Review by sankarpn — APPROVED** [review]
LGTM

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**